client/src/document/ingredients/browser-compatibility-table/index.tsx

```diff
@@ -82,16 +108,29 @@ export function BrowserCompatibilityTable({
 
   const [platforms, browsers] = gatherPlatformsAndBrowsers(category);
 
+  function getNewIssueURL() {
+    const url = "https://github.com/mdn/browser-compat-data/issues/new";
+    const sp = new URLSearchParams();
+    const body = NEW_ISSUE_TEMPLATE.replace(/\$PATHNAME/g, location.pathname)
+      .replace(/\$DATE/g, new Date().toISOString())
+      .replace(/\$QUERY_ID/g, query)
+      .trim();
+    sp.set("body", body);
+    sp.set("title", query);
+    return `${url}?${sp.toString()}`;
+  }
+
   return (
     <BrowserCompatibilityErrorBoundary>
       <BrowserInfoContext.Provider value={browserInfo}>
         {title && <h2 id={id}>{title}</h2>}
         <a
           className="bc-github-link external external-icon"
-          href="https://github.com/mdn/browser-compat-data"
+          href={getNewIssueURL()}
           rel="noopener"
+          title="Report an issue with this compatability data"
         >
-          Update compatibility data on GitHub
+          See something? Say something ...on GitHub
```

Comment:
It's a shame because this is clever (though not widely understood outside of North America—British people don't get it, at least). How about this?

```suggestion
          Report problems with this data on GitHub
```